src/components/Shop/Shop.js

```diff
@@ -57,32 +88,64 @@ function Shop() {
             return (<ListItemText className='titleFilter' primary="Filter by color"/>)
         else return (<ListItemText className='titleFilter' primary="Filter by size"/>)
     }
-
+    
     function Products() {
-        return <div className="shop">
-                    {products && products.map((prod) => (
+        return ( 
+            <>
+            
+        <div className="shop">
+            <div className='row'>
+                
+                <FormControl variant="outlined" className='formControl'>
+                    <Select
+                        labelId="demo-simple-select-label"
+                        id="demo-simple-select"
+                        value={sort}
+                        onChange={handleChange}>
+                        
+                        <MenuItem value={'popularity'}>Sort by Popularity</MenuItem>
```

Comment:
value should be in format `{sort, order}` so it is easier to manipulate with it on backend.